Modules/Tokens/src/Tokens.Infrastructure/Persistence/Database/Repository/TokenRepository.cs

```diff
@@ -1,6 +1,5 @@
-﻿using Backbone.Modules.Tokens.Application.Infrastructure;
+﻿using Backbone.Modules.Tokens.Application.Infrastructure.Persistence.Repository;
```

Comment:
I'm confused. What is imported from this namespace? Shouldn't it be `Backbone.Modules.Tokens.Infrastructure.Persistence.Database.Repository` (note the `Database` fragment)?

Reply:
I think I got confused about the Database directory because it does not exist in other modules.